api/amqp.md

```diff
@@ -0,0 +1,222 @@
+# AMQP API
+
+This document specifies the messages expected by the connector through a publish/subscriber middleware that supports the AMPQ protocol. These messages can be listed in two main categories:
+
+### Southbound traffic (commands):
+
+- Update data
+- Request data
+
+### Northbound traffic (control, measurements):
+
+- Register thing
+- Unregister thing
+- Update thing's schema
+- Publish thing's data
+
+## Message types (Southbound)
+
+### Update data
+
+Updates a thing's sensor value.
+
+#### Topic
```

Comment:
As this is an amqp api document we should use the names defined by the protocol.
In other words:
topic -> exchange
key -> binding key

Reply:
Done.